### DISCUSSION TITLE
Prefered name ?

### DISCUSSION THREAD

**Discussion by pinage404** [gitkraken/gk-cli#69] [General]
There are several names for this package

1. `GitKraken CLI`
	1. in the `README`
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L9
1. `gitkraken-cli`
	1. Brew
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L22
	1. SnapCraft
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L38
	1. Arch User Repository
	https://repology.org/project/gitkraken-cli/versions
1. `gk`
	1. MacPorts
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L27
	1. Release page
	https://github.com/gitkraken/gk-cli/releases/tag/v2.1.2
1. `gitkraken.cli`
	1. WinGet
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L67
1. `gkcli`
	1. Chocolatey
	https://github.com/gitkraken/gk-cli/blob/e4a339e1c5c8eb3176217cebb818e2b4305b9099/README.md?plain=1#L73
1. `gk-cli`
	1. This repository name
	https://github.com/gitkraken/gk-cli
	1. nixpkgs
	https://repology.org/project/gk-cli/versions

---

What is the prefered name ?

---

I maintain the package on `nixpkgs`, should i rename it ?

Having the same name as other packages in https://repology.org would allow to have semi automatic update process on `nixpkgs`

Alternatively, it is possible to merge packages name in [the repository of repology](https://github.com/repology/repology-rules)

**Comment by thenexxuz** [comment]
Hello! I am the maintainer of the AUR package. First off I want to state that I am not employed by GitKraken. I just want to shed some light on why I called the package what I did. I just wanted to have the package name be in line with the other GitKraken AUR packages for GitKraken products otherwise I would have called it "gk-cli" like this repo.
